daniel-alejandro-garcia/tarea-11/to-do-list-routing/src/components/taskEleven.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+export default function TaskEleven() {
```

Comment:
@da-garcia , this is a very bad approach, you should only have one Task component and use props to pass the information